dags/resources/stages/parse/table_definitions/compound/cToken_event_AccrueInterest_combined.sql

```diff
@@ -0,0 +1,24 @@
+-- AccrueInterest for some tokens don't include the cashPrior field. This view combines events with both signatures
+SELECT * except(cashPrior)
```

Comment:
I'm not sure if it'd be better to include this field and populated it with null below

Reply:
It feels like your above option would be good, but would make the sql more complicated? :)

Reply:
Yeah I'd need to list all the columns instead of using `*`

Reply:
Fixed it